### ISSUE TITLE
Problem in core Image.anim_avaible

### ISSUE THREAD

**Issue by krow89** [kivy/kivy#5493]
Hi... I am a Kivy begginer and (more importantly) entusiast... Anyway i found (maybe) a strange behaviour on kivy.core.image.Image class... First some code that illustrate:
```
from kivy.core.image import CoreImage
# ... other code
image= CoreImage.load( 'test_image.gif' )
# In the next line there is the problem... With animated gif, 
# I expect a 'True' but it finish with a printed 'False'
print image.anim_avaible
```
I dont understand the reason behind but, reading some kivy code, i have conclused that need referencing image texture before attr anim_avaible get right value:
```
from kivy.core.image import CoreImage
# ... other code
image= CoreImage.load( 'test_image.gif' )
# Here reference image texture
tex= image.texture
# Now no problem in next line with animated gif... True is returned
print image.anim_avaible
```
Is normal this behaviour? It' s wanted? Thanks

**Comment by KeyWeeUsr** [comment]
It appears to me that it's this line that actually starts setting the property:

https://github.com/kivy/kivy/blob/48418bd22b930640012d686846fe5342f7ad374f/kivy/core/image/__init__.py#L789

which is basically a part of `Image.texture()`. If you call it (and `texture` is a property), it executes that function, thus sets `_anim_available` to `True`. I'm not sure whether the current behavior is correct because the docs appear to me like describing the availability of the animation no matter what (without accessing other object attributes before).

**Comment by krow89** [comment]
Exactly.... I dont think this is the right behaviour (it's really no sense for me and bug prone)
